CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
```

Comment:
Right because sexual harassment is a HUGE problem on Github and this pledge will really help make an impact.

Reply:
1. Sexual harassment is absolutely an issue, especially in a male-dominated field such as this CS. 

2. I realize you're being sarcastic, and I cede that this won't directly impact the problem, but it's important to be there for a few reasons. We can talk offline about those.

3. Why not? It's literally a template that we fill in our emails for and it never comes up again until we want to say "hey, we don't tolerate that here. See the Code of Conduct." Except we can't do that if we don't have a Code of Conduct. It's a pretty standard doc file on most open source projects.